### DISCUSSION TITLE
Accept pull request for Neotest and Overseer?

### DISCUSSION THREAD

**Discussion by olimorris** [rebelot/heirline.nvim#85] [Show and tell]
I use `overseer.nvim` with `neotest` to show the output of my tests in Heirline:

![](https://user-images.githubusercontent.com/9512444/198676197-84da5817-a522-492f-8ce6-4dae912dacf8.gif)

- https://github.com/stevearc/overseer.nvim/discussions/54#discussioncomment-3994528

Would you accept a PR in the cookbook for this?

**Comment by rebelot** [comment]
It would be great if you could put it here :) https://github.com/rebelot/heirline.nvim/wiki/Exotic-recipes

**Reply by olimorris** [reply]
Oh my word. This is great!

**Comment by Zeioth** [comment]
@olimorris your statusline looks great. Consider PR it to [heirline-components.nvim](https://github.com/Zeioth/heirline-components.nvim).
